tests/unit/app/models/test_config.py

```diff
@@ -4117,3 +4119,54 @@ def test_ols_config_with_quota_handlesr_missing_name():
                 },
             }
         )
+
+
+def test_proxy_config_default_values():
+    """Test the ProxyConfig model default values."""
+    proxy_config = ProxyConfig()
+    os_proxy = os.getenv("https_proxy")
+    if os_proxy is None:
+        os_proxy = os.getenv("HTTPS_PROXY")
+    assert proxy_config.proxy_url == os_proxy
```

Comment:
Can you explicitly test the cases when the env is missing and contains a value?

Reply:
new test case added for assignment by env var